app/src/main/java/com/ajdi/yassin/popularmovies/LoadingActivity.java

```diff
@@ -0,0 +1,29 @@
+package com.ajdi.yassin.popularmovies;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.os.Handler;
+import android.widget.ImageView;
+
+
+import com.bumptech.glide.Glide;
+
+public class LoadingActivity extends Activity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_loading);
+        ImageView imageView=(ImageView)findViewById(R.id.image_loading);
+        Glide.with(getBaseContext()).load(R.raw.marvle).into(imageView);
+        startLoading();
+    }
+    private void startLoading() {
+        Handler handler = new Handler();
+        handler.postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                finish();
+            }
+        }, 8000);
```

Comment:
8 seconds is too long. can you reduce it

Reply:
I'll bring a gif that takes a total of 5 seconds and play it for 4 seconds.